man/systemd.service.xml

```diff
@@ -63,6 +63,22 @@
     <para>The <citerefentry><refentrytitle>systemd-run</refentrytitle><manvolnum>1</manvolnum></citerefentry>
     command allows creating <filename>.service</filename> and <filename>.scope</filename> units dynamically
     and transiently from the command line.</para>
+
+    <para>In addition to the various drop-in behaviors described in
+    <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
+    services also support a top-level drop-in with <filename>-.service.d/</filename> that allows
+    modifying the settings of all services on the system. Drop-in files must contain appropriate section headers.
+    The precedence of applying multiple drop-in configurations follows what is defined in
+    <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>.
+    However, configurations in <filename>-.service.d/</filename> have the lowest precedence compared to settings
+    in the service specific override directories. For example, for <filename>foo-bar-baz.service</filename>,
+    drop-ins in <filename>foo-bar-baz.service.d/</filename> override the ones in
+    <filename>foo-bar-.service.d/</filename>, which override the ones <filename>foo-.service.d/</filename>,
+    which override the ones in <filename>-.service.d/</filename>.
+    </para>
+
+    <para>Warning: top-level drop-ins should be used sparingly, as modifying settings for all
+    services on a system can have unintended consequences.</para>
```

Comment:
Heh, I think we've typically stayed away from "dragons be here" type comments in the man pages. Probably you can go without (especially since if they didn't already know this, the unintended consequences might not be obvious, so it just sounds ominous).